lib/dnstap-parser/build.rs

```diff
@@ -0,0 +1,8 @@
+fn main() {
+    println!("cargo:rerun-if-changed=proto/dnstap.proto");
+    let mut prost_build = prost_build::Config::new();
+    prost_build.btree_map(["."]);
+    prost_build
+        .compile_protos(&["proto/dnstap.proto"], &[] as &[&str])
```

Comment:
```suggestion
        .compile_protos(&["proto/dnstap.proto"], &["proto"])
```

I _think_ we can use `proto` here explicitly.